client/src/features/projects/ProjectsApi.ts

```
@@ -128,7 +128,8 @@ export const projectApi = createApi({
             const resultAllProjectData = await Promise.allSettled(projectRequests);
             const projectList = [];
             for (const projectData of resultAllProjectData)
-              if (projectData.status === "fulfilled") projectList.push(formatProjectMetadata(projectData.value.data));
+              if (projectData.status === "fulfilled" && !projectData.value.error)
+                projectList.push(formatProjectMetadata(projectData.value.data));
```

Note: it seems that the `{ error: e as FetchBaseQueryError }` return below is not processed; the error seems to be silently swallowed in `client/src/utils/customHooks/useGetRecentlyVisitedProjects.ts`.